### PR TITLE
ImageAndText

### PR DESCRIPTION
**Pull request by DavidZeLiang**
Add ImageAndText dataset onboarding user guide.
Add sample datasets.

### CODE REVIEW

File: sample/dataset/readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+All data file use comma(,) as delimiter.
+
+### Text Modality
+
+- data column(essential header): text
+- data value: it should be one text without line break, no be Null/NaN/empty.
```
Comment (bowgong) [inline sample/dataset/readme.md:6]: what does this mean?

File: sample/dataset/readme.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- label column(support multiple headers but no duplicate, at leaset one label): hate, violent
```
Comment (bowgong) [inline sample/dataset/readme.md:7]: label column**s**?

Comment (bowgong) [inline sample/dataset/readme.md]: Does `headers` have special meaning? If looking it from CSV perspective, combining header and column may bring confusions, or just my misunderstanding ...

File: sample/dataset/readme.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+All data file use comma(,) as delimiter.
+
+### Text Modality
+
+- data column(essential header): text
```
[the remark this answers is not in the document]
Comment (DavidZeLiang) [reply sample/dataset/readme.md]: fixed

File: sample/dataset/readme.md (hunk 2 above)
Comment (DavidZeLiang) [reply sample/dataset/readme.md]: fixed

File: sample/dataset/readme.md (hunk 1 above)
Comment (DavidZeLiang) [reply sample/dataset/readme.md]: fixed

File: sample/dataset/readme.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+All data file use comma(,) as delimiter.
```
[the remark this answers is not in the document]
Comment (DavidZeLiang) [reply sample/dataset/readme.md:1]: As the base64 string could be too long to read, we use separate csv files.

File: sample/dataset/readme.md (hunk 2 above)
Comment (DavidZeLiang) [reply sample/dataset/readme.md]: It could. All "header" related description are removed.

**Pull request merged**